tests/components/enphase_envoy/snapshots/test_diagnostics.ambr

```diff
@@ -1637,12 +1701,14 @@
               'capabilities': dict({
                 'state_class': 'total_increasing',
               }),
+              'categories': dict({
+              }),
               'config_entry_id': '45a36e55aaddb2007c5f6602e0c38e72',
               'device_class': None,
               'disabled_by': None,
               'domain': 'sensor',
               'entity_category': None,
-              'entity_id': 'sensor.envoy_<<envoyserial>>_lifetime_net_energy_consumption',
+              'entity_id': 'sensor.envoy_<<envoyserial>>_energy',
```

Comment:
The items besides categories should probably not change.

Reply:
Hmm seems like I forgot to update the translations before generating these 🤦

Reply:
fixed in [51f8417](https://github.com/home-assistant/core/pull/110897/commits/51f841770180d7f08dda3ccbdffd6a5fc8d28f94)